client/context.go

```diff
@@ -208,10 +208,10 @@ func (ctx Context) PrintString(str string) error {
 	return err
 }
 
-// PrintOutput outputs toPrint to the ctx.Output based on ctx.OutputFormat which is
+// PrintObject outputs toPrint to the ctx.Output based on ctx.OutputFormat which is
 // either text or json. If text, toPrint will be YAML encoded. Otherwise, toPrint
 // will be JSON encoded using ctx.JSONMarshaler. An error is returned upon failure.
-func (ctx Context) PrintOutput(toPrint proto.Message) error {
+func (ctx Context) PrintObject(toPrint proto.Message) error {
```

Comment:
How about `PrintProtoMessage`? It sounds more like what it does

Reply:
OK, I've already [proposed](https://github.com/cosmos/cosmos-sdk/pull/8107#issuecomment-740265807) `PrintProto` as an alternative. So what do you think about `PrintProto` instead of `PrintProtoMessage`? (we it's shorter and no need to repeat a full type name)

Reply:
That sounds fine :+1:

Reply:
ACK

Reply:
Updated to `PrintProto`.